python/src/aiconfig/default_parsers/openai.py

```diff
@@ -536,3 +581,41 @@ def is_prompt_template(prompt: Prompt):
     return isinstance(prompt.input, str) or (
         hasattr(prompt.input, "data") and isinstance(prompt.input.data, str)
     )
+
+def build_output_data(
+    message: Union[ChatCompletionMessage, None],
+) -> Union[OutputDataWithValue, str, None]:
+    output_data : Union[OutputDataWithValue, str, None] = None
+    if message is not None:
```

Comment:
nit: 
```suggestion
    if not message:
      return None
```

Then everything below doesn't have to be nested in the if

Reply:
Good idea, I'll return early. But still going to keep `message is not None` check because if message is empty string (which is possible) it would pass `if not message`

I know I specify that message can only be of type `Union[ChatCompletionMessage, None]` as the function argument, but Python type hints are unreliable and you can actually pass in whatever you want and the function will still process it